templates/articles/blocks/promoted_list_block.html

```diff
@@ -15,7 +15,7 @@
             {% for item in value.promoted_items %}
             <li class="related-resources__list-item">
                 <h3 class="related-resources__link-heading">
-                    <a class="related-resources__link" href="{{ item.url }}" data-component-name="{{ value.block.meta.label }}: {{ value.heading }}" data-link-type="Links" data-link="{{ item.title }}">
+                    <a class="related-resources__link" href="{{ item.url }}" data-component-name="{{ value.heading }}" data-link-type="Links" data-link="{{ item.title }}">
```

Comment:
Hi Jane, good work, just missing two bits, `data-position` and `Link list` instead of `Links`:

```suggestion
                    <a class="related-resources__link" href="{{ item.url }}" data-component-name="{{ value.heading }}" data-link-type="Link list" data-position="{{ forloop.counter0 }}" data-link="{{ item.title }}">
```

Reply:
Oh thank you James - I totally missed this - updated now